storm-core/src/clj/org/apache/storm/daemon/supervisor.clj

```diff
@@ -1088,8 +1104,26 @@
                                             (str "file:///" storm-log4j2-conf-dir))
                                           storm-log4j2-conf-dir)
                                      file-path-separator "worker.xml")
+
+          cgroup-command (if (conf STORM-CGROUP-ENABLE)
```

Comment:
Maybe it is better to disable cgroup in local mode since the cgroup is not supported in Windows or some linux with old core version